app.js

```diff
@@ -36,62 +17,40 @@ function multiply(num3, num4){
-Write a function called sumArray() that takes in an array of numbers as its single argument and then returns an array where the first element is the sum of the numbers in the array, and the second element is a string that EXACTLY follows this example and uses the values that were input into the function:
-"2,3,4 was passed in as an array of numbers, and 9 is their sum."
-IMPORTANT DETAIL: You may not use the arithmetic operator + in this function. To do addition, use your sum() function that you've already created. You're going to have to be resourceful to figure out how to do this.
-Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testSumArray() function and see if the test passes.*/
-
-// Write your code here
-// var testArray = [2,3,4]; //eslint-disable-line
-
-// function sumArray(testArray){ //eslint-disable-line
 
-
-
-// Here is the test for sumArray(); uncomment it to run it
-
-// testSumArray(testArray);
-
-// Once you get the test passing, do an a-c-p cycle and synchronize the code between GitHub and your laptop. Don't forget to create a new branch for your work on the next question!
-
-/////////////////////////////////////
-/* Problem 5
-Write a function called multiplyArray() that takes an array of numbers as its argument and returns an array whose first element is the product of those numbers, and the second element is a string that EXACTLY follows this example and uses the values that were input into the function:
-"The numbers 2,3,4 have a product of 24."
-IMPORTANT DETAIL: You may not use the arithmetic operator * in this function. To do multiplication, use your multiply() function that you've already created. You're going to have to be resourceful to figure out how to do this.
-Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testMultiplyArray() function and see if the test passes.*/
-
-// Write your code here
-// function multiplyArray(testArray){ //eslint-disable-line
-
-
-
-// Here is the test for multiplyArray(); uncomment it to run it
-// testMultiplyArray(2,3,4);
-
-// Once you get the test passing, do an a-c-p cycle and synchronize the code between GitHub and your laptop. You're done! Submit the link to the repo following the instructions in Canvas.
+function sumAndMultiply(num5, num6, num7){
+  var arraySumult = [];
+  var first = sum(sum(num5, num6)[0], num7)[0];
+  var second = multiply(multiply(num5, num6)[0], num7)[0];
+  arraySumult[0] = first;
+  arraySumult[1] = second;
+  arraySumult[2] = num5 + ' and ' + num6 + ' and ' + num7 + ' sum to ' + first + '.';
+  arraySumult[3] = 'The product of ' + num5 + ' and ' + num6 + ' and ' + num7 + ' is ' + second + '.';
+
+  console.log(arraySumult);
+  return arraySumult;
+}
+testSumAndMultiply(4,7,5);
+
+var testArray = [2,3,4];
+function sumArray(testArray){
+  var arraySumNums = [];
+  var firstElement = sum(sum(2,3)[0], 4)[0];
```

Comment:
You should be adding the values out of the array rather than just plugging them in as static numbers here. So if the values in the array change, this function should still work.